motors/tacho_motor_class.c

```diff
@@ -591,6 +747,9 @@ static ssize_t encoder_polarity_show(struct device *dev,
 {
 	struct tacho_motor_device *tm = to_tacho_motor(dev);
 
+	if (!tm->supports_encoder_polarity)
```

Comment:
Don't want to return an error here. Userland program might depend on this value and it's easier to not have to handle an error. The driver will always return `normal` because it is normal and this OK.